protocol/message.go

```diff
@@ -0,0 +1,50 @@
+package protocol
+
+import "github.com/coniks-sys/coniks-go/merkletree"
+
+// Defines constants representing the types
+// of messages exchanged by clients and servers.
+const (
+	RegistrationType = 0
+	CommitmentType   = 1
```

Comment:
It's just a placeholder for monitoring function. I added it to make this lib align with the java implementation. It looks a little bit awkward if there is no type with value 1.